lib/matplotlib/__init__.py

```diff
@@ -273,9 +273,11 @@ def _get_executable_info(name):
 
     Returns
     -------
-    If the executable is found, a namedtuple with fields ``executable`` (`str`)
-    and ``version`` (`distutils.version.LooseVersion`, or ``None`` if the
-    version cannot be determined).
+
+    tuple or None:
```

Comment:
```suggestion
    tuple or None
```

Reply:
From numpydoc:
`[...]The colon must be preceded by a space, or omitted if the type is absent.` Which, usually is when you have a name for the value and the example n numpydoc:

```
Returns
-------
int
    Description of anonymous integer return value.
```

So I think that no, there is no `:`. I'm happy to put it.

Reply:
Sorry I misread the diff. you are right. I'm agreeing with you.